.travis.yml

```diff
@@ -96,11 +96,10 @@ jobs:
       env: [ TEST_SUITE=build, COVERAGE=true, 'SPEC=mpich' ]
 
 stages:
-  - 'flake8 + documentation'
```

Comment:
why are these commented out?

Reply:
Testing just the mac os part, which only runs as a cron job.  All of that will be reverted once the Mac OS builder passes.